travis.sh

```diff
@@ -153,14 +143,15 @@ function run_xvfb() {
    travis_fold end xvfb
 }
 
-function prepare_catkin_workspace() {
-   travis_fold start catkin.ws "Setting up catkin workspace"
+function prepare_ros_workspace() {
+   travis_fold start ros.ws "Setting up ROS workspace"
    travis_run_simple mkdir -p $ROS_WS/src
    travis_run_simple cd $ROS_WS/src
 
-   # Pull additional packages into the catkin workspace
+   # Pull additional packages into the ros workspace
    travis_run wstool init .
```

Comment:
This should probably be changed to vcstool in the future since it supports both .repos and .rosinstall files

Reply:
Sure. But that is certainly scope creep